dist/index.js

```diff
@@ -254,6 +279,39 @@ var encodeV1 = (key, value) => {
       len.writeUInt16BE(encodedInteractionTypes.byteLength);
       return Buffer.concat([tagBuf, len, encodedInteractionTypes]);
     }
+    case "marketable":
+    case "stables":
+    case "ethTransfers":
+    case "defiSwap": {
+      let encodedHomepagePending = Buffer.allocUnsafe(0);
+      Object.entries(value).forEach(([key2, value2]) => {
+        const encoded = encodeV1(key2, value2);
+        if (encoded) {
+          encodedHomepagePending = Buffer.concat([
+            encodedHomepagePending,
+            encoded
+          ]);
+        }
+      });
+      const len = Buffer.allocUnsafe(2);
+      len.writeUInt16BE(encodedHomepagePending.byteLength);
+      return Buffer.concat([tagBuf, len, encodedHomepagePending]);
+    }
+    case "optimisticL2": {
```

Comment:
Just going to point out again that the code blocks for defiSwap and optimisticL2 are the same in case we want to merge those cases.

Reply:
Thats true in the dist but in the ts file they use different types

Reply:
ah ok- the dist!